lib/ocpp/common/websocket/websocket_libwebsockets.cpp

```diff
@@ -638,6 +650,15 @@ void WebsocketLibwebsockets::client_loop() {
 
     auto& uri = this->connection_options.csms_uri;
 
+    std::string ocpp_versions;
+    for (bool first = true; auto version : this->connection_options.ocpp_versions) {
```

Comment:
Range based for loops with initializer is only available in C++20, we are still on C++17.

Reply:
Yeah I saw, didn't know this and my compiler just worked. I'll fix it ;)

Reply:
```suggestion
    bool first = true;
    for (auto version : this->connection_options.ocpp_versions) {
```